pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1846,6 +1847,13 @@ func buildListener(opts buildListenerOpts) *xdsapi.Listener {
 		FilterChains:    filterChains,
 		DeprecatedV1:    deprecatedV1,
 	}
+
+	if util.IsProtocolSniffingEnabledForNode(node) {
+		listener.ListenerFiltersTimeout = &features.ProtocolDetectionTimeout
+		listener.ContinueOnListenerFiltersTimeout = true
```

Comment:
no such field?

Reply:
Yes. It will be introduced in the new proxy.

Reply:
https://github.com/envoyproxy/envoy/pull/7859